editor/src/components/editor/store/editor-state.ts

```diff
@@ -1010,6 +1019,20 @@ export function githubRepo(owner: string, repository: string): GithubRepo {
   }
 }
 
+function githubRepoName(repo: GithubRepo): string {
+  return `${repo.owner}/${repo.repository}`
+}
+
+export function githubRepoEquals(a: GithubRepo | null, b: GithubRepo | null): boolean {
+  if (a == null && b == null) {
+    return true
+  }
+  if (a == null || b == null) {
+    return false
+  }
+  return githubRepoName(a) === githubRepoName(b)
```

Comment:
It would be more preferable to check the two fields `owner` and `repository` against each other here, as `githubRepoName` is making 2 new strings for each comparison.

Reply:
https://github.com/concrete-utopia/utopia/pull/2762/commits/a542d5a15696df0a91dbe111ae58fca53c3bc46f